cmd/cstorvolumeclaim/controller.go

```diff
@@ -188,7 +180,7 @@ func (c *Controller) create(cvc *apis.CStorVolumeClaim) error {
 
 // UpdateCVCStatus updates the status block of the CVC resource to reflect the
 // current state of the world
-func (c *Controller) updateCVCStatus(cvc *apis.CStorVolumeClaim,
+func (c *CVCController) updateCVCStatus(cvc *apis.CStorVolumeClaim,
```

Comment:
we want to name this as updateCVC right?

Reply:
No, we can't have updateCVC as we already using the same name for update sync